hugo_site/content/hospitality.md

```diff
@@ -56,7 +56,7 @@ Date range: From 10th of April to the 14th of April
 
 ## Danhostel
 
-[www.danhostel.dk](https://www.danhostel.dk/)
+[www.cphhostel.dk](https://www.cphhostel.dk/en)
```

Comment:
Heh, I can nitpick this a bit :smile: The correct link is actually `https://danhostelcopenhagencity.dk/en` - technical reasoning: No unencrypted connection will be established in order to forward the user's request.